### PR TITLE
Improve Prom exporter docs

### PR DESCRIPTION
**Pull request by thelabdude**
Add instructions for using the Prometheus exporter with Prometheus stack

### CODE REVIEW

File: docs/solr-prometheus-exporter/README.md [hunk 1]
```diff
@@ -49,4 +49,188 @@ All ACL fields are **required** if an ACL is used.
...
+```
+kubectl get svc dev-prom-exporter-solr-metrics --output jsonpath="{@.spec.ports}"
+```
+You should see output similar to:
+```
+[{"name":"solr-metrics","port":80,"protocol":"TCP","targetPort":8080}]
+```
+
+### Create a Service Monitor
+The Prometheus operator (deployed with the Prometheus stack) uses service monitors to find which services to scrape metrics from. Thus, we need to define a service monitor for our exporter service `dev-prom-exporter-solr-metrics`.
```
Comment (HoustonPutman) [inline docs/solr-prometheus-exporter/README.md:189]: I would add a comment that this differs from the Kubernetes discovery configuration provided by base prometheus: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#service

**Review by HoustonPutman — APPROVED** [review]
This is awesome stuff!!

It would be great in the future when we have a prometheus-exporter helm chart, to be able to auto-install the serviceMonitor for users as well (if desired).

File: docs/solr-prometheus-exporter/README.md [hunk 2]
```diff
@@ -49,4 +49,188 @@ All ACL fields are **required** if an ACL is used.
...
+GRAFANA_POD_ID=$(kubectl get pod -l app.kubernetes.io/name=grafana --no-headers -o custom-columns=":metadata.name" -n monitoring)
+kubectl port-forward -n monitoring $GRAFANA_POD_ID 3000
+```
+Open Grafana using `localhost:3000` and login with username `admin` and password `prom-operator`.
+
+Once logged into Grafana, import the Solr dashboard JSON corresponding to the version of Solr you're running.
+
+Solr does not export any useful metrics until you have at least one collection defined.
+
+_Note: Solr 8.8.0 includes an updated dashboard that includes better metrics for query performance._
```
Comment (anshumg) [inline docs/solr-prometheus-exporter/README.md:236]: May be switch this to `Solr 8.8.0 and newer versions`

**Review by anshumg — APPROVED** [review]
This looks great! Thanks @thelabdude .

**Pull request merged**